GdkPixbuf-2.0.gir

```diff
@@ -670,11 +823,8 @@ exact given width. When not preserving aspect ratio, a @width or
 @height of -1 means to not scale the image at all in that dimension.
 
 The stream is not closed.</doc>
-        <return-value transfer-ownership="full">
-          <doc xml:space="preserve">A newly-created pixbuf, or %NULL if any of several error
-conditions occurred: the file could not be opened, the image format is
-not supported, there was not enough memory to allocate the image buffer,
-the stream contained invalid data, or the operation was cancelled.</doc>
+        <return-value transfer-ownership="full" nullable="1">
```

Comment:
This needs some overriding, see https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/merge_requests/105#note_1277108

Unfortunately for `gdk_pixbuf_animation_new_from_stream_async()` overriding it has no effect because of a bug in gir, so that needs to be fixed first.

Reply:
https://github.com/gtk-rs/gir/pull/1227